1-js/02-first-steps/01-hello-world/article.md

```diff
@@ -1,87 +1,86 @@
 
-The `language` attribute: <code>&lt;script <u>language</u>=...&gt;</code>
-: This attribute was meant to show the language of the script. This attribute no longer makes sense because JavaScript is the default language. There is no need to use it.
+`language` এট্রিবিউটসটি: <code>&lt;script <u>language</u>=...&gt;</code>
+: এই এট্রিবিউটসটি স্ক্রিপ্টের ভাষা দেখানোর জন্য বুঝানো হয়েছে।  এই এট্রিবিউটসটি দিয়ে এখন আর কিছু হয় না কারন জাভাস্ক্রিপ্তটি এখন ডিফল্ট ভাষা। এটা ব্যবহার করার এখন আর প্রয়োজন নেই।
 
-Comments before and after scripts.
-: In really ancient books and guides, you may find comments inside `<script>` tags, like this:
+স্ক্রিপ্টগুলির আগে এবং পরে মন্তব্যগুলিঃ 
+সত্যিই প্রাচীন বই এবং গাইডগুলিতে, আপনারা হয়তো `<script>` ট্যাগগুলোর মধ্যে মন্তব্যগুলি পেতে পারেন, এটার মতঃ
 
-    ```html no-beautify
+    ```এইচটিএমএল বিউটিফাই করা নাই
     <script type="text/javascript"><!--
         ...
     //--></script>
     ```
+    
+    আধুনিক জাভাস্ক্রিপ্টে এই কৌশলটি আর ব্যবহার করা হয় না। এই মন্তব্যগুলি জাভাস্কিপ্ট কোডগুলো কে লুকিয়ে রাখে পুরনো ব্রাউজার গুলো থেকে যারা জানে না কিভাবে `<script>` ট্যাগকে প্রসেস করতে হয়।
+    যেহেতু গত 15 বছরে প্রকাশিত ব্রাউজারগুলিতে এই সমস্যাটি নেই, এই জাতীয় মন্তব্য আপনাকে সত্যিকারের পুরানো কোড সনাক্ত করতে সহায়তা করতে পারে।
 
-    This trick isn't used in modern JavaScript. These comments hide JavaScript code from old browsers that didn't know how to process the `<script>` tag. Since browsers released in the last 15 years don't have this issue, this kind of comment can help you identify really old code.
 
+## এক্সটার্নাল স্ক্রিপ্ট
 
-## External scripts
+যদি আমাদের অনেক জাভাস্কিপ্ট কোড থাকে, তাহলে আমরা এইগুলোকে আলাদা ফাইলে রাখতে পারি।
 
-If we have a lot of JavaScript code, we can put it into a separate file.
-
-Script files are attached to HTML with the `src` attribute:
+স্কিপ্ট ফাইল গুলো এইচটিএমএল এ সংযুক্ত করা হয় `src` এট্রিবিউটস দিয়েঃ 
 
 ```html
 <script src="/path/to/script.js"></script>
 ```
 
-Here, `/path/to/script.js` is an absolute path to the script from the site root. One can also provide a relative path from the current page. For instance, `src="script.js"` would mean a file `"script.js"` in the current folder.
+এখানে, `/path/to/script.js` এই পথটি হয় সাইট এর মূল থেকে স্ক্রিপ্টের জন্য একটি সঠিক পথ। বর্তমান পৃষ্ঠা থেকে যে কেউ আপেক্ষিক পথ সরবরাহ করতে পারে. উদাহরণস্বরূপ, `src="script.js"` এই `"script.js"` ফাইল দিয়ে বর্তমান ফোল্ডার থেকে বুঝায়। 
 
-We can give a full URL as well. For instance:
+আমরা পুরো URL টিও দিতে পারি। উদাহরণস্বরূপ:
 
 ```html
 <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.2.0/lodash.js"></script>
 ```
 
-To attach several scripts, use multiple tags:
+বহু ট্যাগ ব্যবহার করে, বিভিন্ন স্ক্রিপ্টগুলো সংযুক্ত করতে পারিঃ
```

Comment:
বহু ট্যাগ ব্যবহার করে, বিভিন্ন স্ক্রিপ্টগুলো সংযুক্ত করতে পারিঃ

should be like: -
একাধিক স্ক্রিপ্ট ট্যাগ ব্যবহার করে একাধিক স্ক্রিপ্ট ফাইল সংযুক্ত করতে পারি।